docs/test.ts

```diff
@@ -473,6 +473,26 @@ tracer.appsec.trackUserLoginFailureEvent('user_id', false, meta)
 tracer.appsec.trackCustomEvent('event_name')
 tracer.appsec.trackCustomEvent('event_name', meta)
 
+tracer.appsec.v2.trackUserLoginSuccess("login")
```

Comment:
you're using a mix of double and single quotes, this should probably be a lint rule one day

Reply:
> this should probably be a lint rule one day

Yes there is a big mix of `"` and `'` in this file, replacing this one, thanks.